docs/src/main/asciidoc/security-authentication-mechanisms.adoc

```diff
@@ -779,6 +780,54 @@ quarkus.http.auth.permission.roles1.methods=GET <2>
 <2> Make the `roles1` permission match only the endpoint annotated with the `@AuthorizationCodeFlow` annotation.
 Unannotated endpoints must avoid the delay caused by the `applies-to=JAXRS` option.
 
+=== Use inclusive authentication to enable path-based authentication
+
+By default, Quarkus only supports <<http-security-policy-path-based-auth,path-based authentication>> for one authentication mechanism per path.
```

Comment:
There is also an annotation based path-based authentication...
In https://quarkus.io/guides/security-authentication-mechanisms#combining-authentication-mechanisms, I propose to restructure it a little bit, add `=== Inclusive Authentication` subsection where we talk about it, then `=== Path-based authentication` (which is what we should link to from here), and add `==== Use HttpSecurityPolicy` and `==== Use annotations`.

IMHO that can be a bit better